Gopkg.toml

```diff
@@ -50,8 +50,8 @@
   version = "1.0.1"
 
 [[constraint]]
+  branch = "master"
   name = "github.com/spf13/cobra"
-  version = "0.0.1"
```

Comment:
Down the road it looks like COBRA will move to regular SemVer: https://github.com/spf13/cobra/releases

Are we agreed that it's best to track `master` until then?

Reply:
Yep that's the plan. If we don't track master it doesn't build.